### PR TITLE
Add `.editorconfig`

### PR DESCRIPTION
**Pull request by abouthiroppy**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = false
```
Comment (dleitee) [inline .editorconfig]: @abouthiroppy You can change to 'true'?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
```
Comment (dleitee) [inline .editorconfig]: @abouthiroppy You can add 'trim_trailing_whitespace = true'?

**Comment by abouthiroppy** [comment]
@dleitee updated 😄

**Comment by dleitee** [comment]
@abouthiroppy Thank you!!  😃

**Pull request merged**